### PR TITLE
doc: add registered trademark to Bluetooth mentions

### PR DESCRIPTION
**Pull request by wiba-nordic**
Add several registered trademark symbols to the documentation.
This is to align with the official Bluetooth branding guidelines.
JIRA: https://projecttools.nordicsemi.no/jira/browse/NCSDK-10721

Signed-off-by: Wille Backman <wille.backman@nordicsemi.no>

### CODE REVIEW

File: samples/bluetooth/peripheral_cts_client/README.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52dk_nrf52832
 
-The sample also requires a device running a CTS Server to connect with (for example, a Bluetooth Low Energy dongle and nRF Connect for Desktop)
+The sample also requires a device running a CTS Server to connect with (for example, a Bluetooth® Low Energy dongle and nRF Connect for Desktop)
```
Comment (peknis) [inline samples/bluetooth/peripheral_cts_client/README.rst:26]:
```suggestion
The sample also requires a device running a CTS Server to connect with (for example, a Bluetooth® Low Energy dongle and nRF Connect for Desktop).
```

File: samples/bluetooth/central_smp_client/README.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns , nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf52dk_nrf52832
```
Comment (peknis) [inline samples/bluetooth/central_smp_client/README.rst:33]:
```suggestion
   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf52dk_nrf52832
```

File: doc/nrf/ug_nfc.rst [hunk 3]
```diff
@@ -8,7 +8,7 @@ Near Field Communication (NFC)
    :depth: 2
 
 Near Field Communication (NFC) is a technology for wireless transfer of small amounts of data between two devices.
-It uses very simple communication protocols and therefore allows quicker connection than Bluetooth Low Energy.
+It uses very simple communication protocols and therefore allows quicker connection than Bluetooth® Low Energy.
```
Comment (peknis) [inline doc/nrf/ug_nfc.rst:11]:
```suggestion
It uses very simple communication protocols, and therefore, allows quicker connection than Bluetooth® Low Energy.
```

File: samples/bluetooth/central_hr_coded/README.rst [hunk 4]
```diff
@@ -13,7 +13,7 @@ However, this sample specifically looks for heart rate monitors using LE Coded P
 Overview
 ********
 
-The sample demonstrates a Bluetooth LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support which not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
+The sample demonstrates a Bluetooth® LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support which not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
```
Comment (peknis) [inline samples/bluetooth/central_hr_coded/README.rst:16]:
```suggestion
The sample demonstrates a Bluetooth® LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support, which not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
```

File: samples/bluetooth/peripheral_ancs_client/README.rst [hunk 5]
```diff
@@ -32,7 +32,7 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52dk_nrf52832
 
-The sample also requires a device running an ANCS Server to connect with (for example, an iPhone which runs iOS, or a Bluetooth Low Energy dongle and nRF Connect for Desktop).
+The sample also requires a device running an ANCS Server to connect with (for example, an iPhone which runs iOS, or a Bluetooth® Low Energy dongle and nRF Connect for Desktop).
```
Comment (peknis) [inline samples/bluetooth/peripheral_ancs_client/README.rst:35]:
```suggestion
The sample also requires a device running an ANCS Server to connect with (for example, an iPhone that runs iOS, or a Bluetooth® Low Energy dongle and nRF Connect for Desktop).
```

File: samples/bluetooth/peripheral_hr_coded/README.rst [hunk 6]
```diff
@@ -13,7 +13,7 @@ However, this sample supports LE Coded PHY.
 Overview
 ********
 
-The sample demonstrates a basic Bluetooth LE Peripheral role functionality that exposes the Heart Rate GATT Service with LE Coded PHY support which not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
+The sample demonstrates a basic Bluetooth® Low Energy Peripheral role functionality that exposes the Heart Rate GATT Service with LE Coded PHY support which not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
```
Comment (peknis) [inline samples/bluetooth/peripheral_hr_coded/README.rst:16]:
```suggestion
The sample demonstrates a basic Bluetooth® Low Energy Peripheral role functionality that exposes the Heart Rate GATT Service with LE Coded PHY support, which is not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by hubertmis — APPROVED** [review]

File: applications/nrf_desktop/README.rst [hunk 7]
```diff
@@ -7,7 +7,7 @@ nRF Desktop
    :local:
    :depth: 2
 
-The nRF Desktop is a reference design of a Human Interface Device (HID) that is connected to a host through Bluetooth LE or USB, or both.
+The nRF Desktop is a reference design of a Human Interface Device (HID) that is connected to a host through Bluetooth® Low Energy or USB, or both.
```
Comment (joerchan) [inline applications/nrf_desktop/README.rst:10]: Any reason why you choose to expand the LE abbreviation here but not other places?

**Review by trond-snekvik — COMMENTED** [review]
To ensure this in all future changes, could this perhaps be an alias, defined in some central location?
ie

```rst
.. |bluetooth| replace:: Bluetooth®
```

then, use that alias everywhere instead of manually spelling out "Bluetooth®"?

File: applications/nrf_desktop/README.rst (hunk 7 above)
Comment (wiba-nordic) [reply applications/nrf_desktop/README.rst:10]: Mainly the fact that HID was also expanded in the same sentence, so it felt natural to expand LE as well.

**Comment by wiba-nordic** [comment]
An alias could be defined to make it easier to produce the registered trademark symbol when using the Bluetooth wordmark. However, the branding guidelines don't require the symbol in all mentions of Bluetooth, only the first and most prominent one. In that sense, it is not strictly necessary since it generally only needs to be used just once per document.

I did use it more than once in a couple cases, mainly when directly referencing the Bluetooth Core Specification, and in known issues since it's harder to know which issues a reader is going to see.

**Review by tomchy — APPROVED** [review]

**Review by trond-snekvik — APPROVED** [review]

File: doc/nrf/ug_matter_configuring.rst [hunk 8]
```diff
@@ -15,7 +15,7 @@ To pair and control the Matter accessory device remotely over such a network, yo
 A Matter controller is a role within the Matter environment.
 This controller interacts with the accessory devices using the following protocols:
 
-* Bluetooth LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network.
+* Bluetoot® LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network.
```
Comment (Avalei) [inline doc/nrf/ug_matter_configuring.rst:18]:
```suggestion
* Bluetooth® LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network.
```

**Review by Avalei — CHANGES_REQUESTED** [review]
There were a few files that had the trademark missing on the very first instance of the term:

applications/nrf_desktop/doc/hid_forward.rst
- Add first trademark to line 12 instead of line 13

 applications/nrf_desktop/doc/hids.rst
- Add first trademark to line 11 instead of line 29

cripts/hid_configurator/README.rst
- Add first trademark to line 25 instead of line 81

I would also be in favor of adding the trademarked term as an alias to create a streamlined approach for consistency in future documentation. It would be one less rule we have to remember.

**Review by sebastiandraus — APPROVED** [review]

**Comment by wiba-nordic** [comment]
Can I get opinions from the reviewers on whether they would prefer having a |bluetooth| alias to use for every instance of Bluetooth in documentation, or having the ® on the first or most prominent instance of the word?

**Comment by joerchan** [comment]
+1 on having it as an alias.

**Comment by Avalei** [comment]
+1 on using the alias.

**Comment by wiba-nordic** [comment]
One of the other writers noted that, since the documentation is public on github and github doesn't process aliases, it's better to keep the symbol. It's the same reason we moved away from using an alias for Bluetooth LE.

**Review by Avalei — APPROVED** [review]

**Pull request merged**